presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -848,6 +853,16 @@ public void addQueryAccessControlInfo(AccessControlInfo accessControlInfo)
         accessControlReferences.setQueryAccessControlInfo(accessControlInfo);
     }
 
+    public void addViewDefinitionReference(QualifiedObjectName viewDefinitionName, ViewDefinition viewDefinition)
+    {
+        viewTextDefinitions.put(viewDefinitionName, viewDefinition);
```

Comment:
I would add these directly to the accessControlReference instead of having to make a separate call at the end.